draft-irtf-cfrg-spake2.xml

```diff
@@ -421,12 +380,23 @@ seed: edwards448 point generation seed (N)
         Hudson also suggested the addition of warnings on the reuse of x and y. Thanks
         to Fedor Brunner, Adam Langley, and the members of the CFRG for
         comments and advice. Chris Wood contributed substantial text and reformatting
-        to address the excellent review comments from Kenny Paterson.
-        Trevor Perrin informed me of SPAKE2+.</t>
+        to address the excellent review comments from Kenny Paterson.</t>
     </section>
   </middle>
   <back>
     <references title="Normative References">
+      <reference anchor="h2c">
+	<front>
+          <title>Hashing To Elliptic Curves</title>
+	  <author initials="A" surname="Faz-Hernandez" />
+	  <author initials="S" surname = "Scott" />
+	  <author initials = "N" surname = "Sullivan" />
+	  <author initials = "R" surname = "Wahby" />
+	  <author initials = "C" surname = "Wood" />
+	  <date month = 'November' year = '2019'/>
+        </front>
+        <seriesInfo name = 'Work in Progress' value='draft-irtf-cfrg-hash-to-curve-05' />
+      </reference>
```

Comment:
You should be able to import an ENTITY from xml.resource.org instead of putting this in by hand, as is already done for uks.  (They update nightly, maybe, IIRC.  Or maybe weekly.)